docs/self-managed/components/components-upgrade/880-to-890.md

```diff
@@ -33,3 +33,128 @@ The default layout displays **time only**, thread name, MDC context, log level,
 ```
 
 See [Logging](/self-managed/components/modeler/web-modeler/configuration/logging.md) for advanced settings and more details.
+
+### Embedded web server
+
+Web Modeler now uses [Apache Tomcat](https://tomcat.apache.org/) as an embedded web server instead of [Undertow](https://undertow.io/).
+
+If you are using a custom Undertow configuration, you will need to migrate it.
+
+You should consider the following when upgrading:
+
+:::info
+
+- For a full list of Spring properties, refer to the [Spring documentation](https://docs.spring.io/spring-boot/appendix/application-properties/index.html).
+- For more information, refer to the [Spring embededded web server documentation](https://docs.spring.io/spring-boot/how-to/webserver.html).
+
+:::
+
+#### Undertow-specific properties (remove/replace)
+
+If you have anything defined under:
+
+```yaml
+server.undertow.*
+```
+
+These properties will not work once you switch to Tomcat. See below for a complete list of important properties, and the recommended action to take.
```

Comment:
🔧 Let's not claim that this is a "complete list" but just highlights the most important properties from Camunda's perspective. Additionally, I'd reference https://docs.spring.io/spring-boot/3.5/how-to/webserver.html#howto.webserver.configure and/or https://docs.spring.io/spring-boot/3.5/appendix/application-properties/index.html#appendix.application-properties.server here.